Testscripts/Linux/ETHTOOL-CHECK-CHANNEL.sh

```diff
@@ -52,42 +51,34 @@ fi
 
 # Get number of channels
 channels=$(ethtool -l "${SYNTH_NET_INTERFACES[@]}" | grep "Combined" | sed -n 2p | grep -o '[0-9]*')
+# Get max number of channels
+max_channels=$(ethtool -l "${SYNTH_NET_INTERFACES[@]}" | grep "Combined" | sed -n 1p | grep -o '[0-9]*')
```

Comment:
aa little bit of duplicate code here: ethtool -l "${SYNTH_NET_INTERFACES[@]}" | grep "Combined"